apps/progs.h

```diff
@@ -2,7 +2,7 @@
  * WARNING: do not edit!
  * Generated by apps/progs.pl
  *
- * Copyright 1995-2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1995-2017 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Do we change the year just for the sake of changing it, or do we change it whenever something else changes in the file?  If the former, then fine, but if the latter, then this change should be removed, since the last commit that actually changes `apps/progs,h` was made in 2016 (05c16493c9fb445a1ccec947de903586503d776c).